services/shhext/api_test.go

```diff
@@ -0,0 +1,42 @@
+package shhext
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMessagesRequest_setDefaults(t *testing.T) {
```

Comment:
There is already a similar test called `TestRequestMessagesDefaults`. I think it can be removed.

Reply:
> What's gonna happen if From is set to 0? To should not be 0 ever.
`To` is never `0`. If it's `!= 0` in the previous `if` is fine, otherwise is set to `0`.
```
if r.To == 0 {
	r.To = uint32(now.UTC().Unix())
}
```

If `From` is 0, then it's `To - 24 hours`. If the result is less then `0` then it should just be `0`. But this last part need to be fixed

Reply:
Ok, that sounds good. In theory, we could allow `From` to be `0` as we will allow it to be `1` but this approach should work better as we don't unnecessary process all requests if someone sends a request without `To` and `From` which is likely.